lib/beaker/dsl/helpers/puppet_helpers.rb

```diff
@@ -714,35 +716,51 @@ def wait_for_host_in_dashboard(host)
 
         # Ensure the host has requested a cert, then sign it
         #
-        # @param [Host, Array<Host>, String, Symbol] host    One or more hosts to act upon,
+        # @param [Host, Array<Host>, String, Symbol] host   One or more hosts to validate signing certificate,
         #                            or a role (String or Symbol) that identifies one or more hosts.
+        #                            No argument, or an empty array means no validation of success
+        #                            for specific hosts will be performed.
         #
         # @return nil
         # @raise [FailTest] if process times out
-        def sign_certificate_for(host)
-          block_on host do | host |
-            if [master, dashboard, database].include? host
+        def sign_certificate_for(host = [])
```

Comment:
Yeah, this method originally took a single host, issued a command to the master to sign all certs, and then validated that a certificate for that single host exists.  I changed it so that you could pass an array of hosts and that it would sign all certs and then validate that certs for all of the hosts were signed.  If you pass nothing in, it will execute the command to sign all certs, but do no validation that any specific hosts are included in the list of certs.